### PR TITLE
[#154442946] transaction details

### PR DESCRIPTION
**Pull request by fgiobergia**
PR for [154442946](https://www.pivotaltracker.com/story/show/154442946), which introduces TransactionDetailsScreen, a screen accessible by tapping on any given transaction and that shows further details about it. 

As usual, the credit card component is a cropped image that will be replaced in [157422715](https://www.pivotaltracker.com/story/show/157422715)

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Remember to fix the PR title by removing WIP wording when ready
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#154442946](https://www.pivotaltracker.com/story/show/154442946): Come CIT devo poter vedere il dettaglio di ciascuna transazione (UI)

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/screens/wallet/TransactionDetailsScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        topContent={topContent}
+      >
+        <Content style={WalletStyles.whiteContent}>
+          <Grid>
+            <Row>
+              <Left>
+                <Text>{`${I18n.t("wallet.total")} ${operation.currency}`}</Text>
+              </Left>
+              <Right>
+                <Text style={WalletStyles.boldStyle}>{operation.amount}</Text>
```
Comment (cloudify) [inline ts/screens/wallet/TransactionDetailsScreen.tsx]: is this a different "bold" then the rest of the app?

**Comment by cloudify** [comment]
looks good so far, remove the `WIP` tag when it's ready

File: ts/components/wallet/TransactionsList.tsx [hunk 2]
```diff
@@ -86,7 +87,13 @@ export class TransactionsList extends React.Component<Props, State> {
             removeClippedSubviews={false}
             dataArray={ops as any[]} // tslint:disable-line
             renderRow={(item): React.ReactElement<any> => (
-              <ListItem onPress={(): boolean => navigate("")}>
+              <ListItem
+                onPress={(): boolean =>
```
Comment (gunzip) [inline ts/components/wallet/TransactionsList.tsx]: @fpersico is it good to trigger navigation from the component or we'd better use a generic action that gets handled in a saga ?

Comment (fpersico) [reply ts/components/wallet/TransactionsList.tsx]: In this case is good. If you have to check something or perform any side effect before navigation using saga is better.

File: ts/screens/wallet/TransactionDetailsScreen.tsx (hunk 1 above)
Comment (fgiobergia) [reply ts/screens/wallet/TransactionDetailsScreen.tsx]: It isn't, but iinm there isn't a "bold" definition in `Text`. Can you have a look at my [latest commit](https://github.com/teamdigitale/italia-app/pull/162/commits/f7d740d8beae97156dafee5cf59b95e5d05058f0) to see whether that's more like what you were expecting?  

Other styles in `ts/components/styles/wallet.ts` are setting `fontWeight` independently, but I'm planning on fixing that as part of [157769361](https://www.pivotaltracker.com/story/show/157769361)

**Comment by fpersico** [comment]
Can you test what happens if any of the strings in the Right of the Row is a bit longer. For example `Comune di Gallarate` becomes `A string a little longer to test the layout"

**Comment by fgiobergia** [comment]
@fpersico the following happens

**Comment by cloudify** [comment]
@matteodesanti what do you think about that? https://github.com/teamdigitale/italia-app/pull/162#issuecomment-391625812

File: ts/screens/wallet/TransactionDetailsScreen.tsx (hunk 1 above)
Comment (fgiobergia) [reply ts/screens/wallet/TransactionDetailsScreen.tsx]: @cloudify any feedback about this?

Comment (cloudify) [reply ts/screens/wallet/TransactionDetailsScreen.tsx]: @fpersico

**Comment by cloudify** [comment]
@fgiobergia I'd say let's go ahead with this PR, can you please add a story for fixing the layout issue with long descriptions? please assign the issue to @matteodesanti and attach the screenshot above

**Comment by matteodesanti** [comment]
@fgiobergia @cloudify story updated with fixed layout.

**Pull request merged**